src/react/hooks/useMarketSeed.tsx

```diff
@@ -0,0 +1,18 @@
+import { defaultMarketPlace as defaultMarketplace } from "consts";
+import { useParams } from "react-router-dom";
+
+export const useMarketSeed = () => {
+	const params = useParams();
+	const storedMarketplace = sessionStorage.getItem("marketplace");
+	const paramMarketplace =
+		(params.marketplace !== "help" && params.marketplace !== "credix-pass" && params.marketplace) ||
```

Comment:
maybe a consts array of these somewhere `NON_MARKETPLACE_ROUTES`? So if we need to add one in the future that it is clear, just a thought